include/debug/cpu_load.h

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+#ifndef __CPU_LOAD_H
+#define __CPU_LOAD_H
+
+#include <zephyr/types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @defgroup cpu_load CPU load
+ * @brief Module for measuring CPU load.
+ *
+ * @{
+ */
+
+/** @brief Initialize CPU load measurement module.
+ *
+ * During initialization TIMER driver and PPI channels are allocated.
+ *
+ * @return 0 on success, error code otherwise.
+ */
+int cpu_load_init(void);
+
+/** @brief Reset measurement. */
```

Comment:
```suggestion
/** @brief Reset the CPU load measurement module. */
```
I guess this can be called only after intialization? Can we mention it here in the documentation? Are the CPU load measurement values reset a spart of this function? Are the previous values stored somewhere on reset?

Reply:
its not module reset but measurement reset. So suggestion is not valid.